packages/waf/lib/waf.ts

```diff
@@ -76,6 +77,21 @@ export interface WebApplicationFirewallProps {
    * Priority numbers must be equal to or bigger than 30
    */
   postProcessCustomRules?: aws_wafv2.CfnWebACL.RuleProperty[];
+
+  /**
+   * Enable CloudWatch logging. Default: false
+   */
+  enableLogging?: boolean;
```

Comment:
Why not set this as true by default? I feel like logging as a default is a good thing

Reply:
Just to keep the status quo by default in terms of cost, I thought. But yeah that sounds better, as the stacks that are using older versions would need to bump up the construct version for future changes anyways. 
Updated.